deploy/helm/templates/swo-agent-statefulset.yaml

```diff
@@ -50,6 +50,9 @@ spec:
                 secretKeyRef:
                   name: {{ template "common.secret" . }}
                   key: SOLARWINDS_API_TOKEN
+          envFrom:
+            - configMapRef:
+                name: {{ include "common.fullname" (tuple . "-common-env") }}
```

Comment:
This will add also variables like `HTTPS_PROXY`, which could have other meaning for the SWO Agent. Is that expected?

Reply:
removed, discussed offline